core/server/lib/common/i18n.js

```diff
@@ -8,10 +8,22 @@ var supportedLocales = ['en'],
     MessageFormat = require('intl-messageformat'),
     logging = require('./logging'),
     errors = require('./errors'),
+    jp = require('jsonpath'),
+    settingsCache = require('../../services/settings/cache'),
 
-    // TODO: fetch this dynamically based on overall blog settings (`key = "default_locale"`) in the `settings` table
-    currentLocale = 'en',
-    blos,
+    // currentLocale, dynamically based on overall settings (key = "default_locale") in the settings db table
+    // (during Ghost's initialization, settings available inside i18n functions below; see core/server/index.js)
+    //
+    // E.g.: en = English (default), es = Spanish, en-US = American English, etc.
+    // Standard:
+    // Language tags in HTML and XML
+    // https://www.w3.org/International/articles/language-tags/
+    //
+    // The corresponding translation files should be at content/themes/mytheme/locales/es.json, etc.
+    currentLocale,
+    activeTheme,
+    coreStrings,
```

Comment:
👍 Like the renaming from `blob` to something more specific.

Reply:
:+1: